.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
None of the double quotes in this file are needed.

seq-in-map lines (starting with `-`) should not be extra indented.